conf/app.routes

```diff
@@ -2,8 +2,6 @@
 
 ->          /hmrc-frontend                               hmrcfrontend.Routes
 
-GET         /                                            controllers.IndexController.onPageLoad
```

Comment:
I think we want the IndexController left in because of the ticket DLS-10353. This change looks like it would break that fix potentially